Makefile

```diff
@@ -65,6 +65,6 @@ maintainer-clean: distclean
 # 	  EXIT_CODE=$$?; kill $$PID; exit $$EXIT_CODE
 
 docs: install-dev
-	$(VENV)/bin/sphinx-build -b html -d $(SPHINX_BUILDDIR)/doctrees docs $(SPHINX_BUILDDIR)/html
+	$(VENV)/bin/sphinx-build -a -n -b html -d $(SPHINX_BUILDDIR)/doctrees docs $(SPHINX_BUILDDIR)/html
```

Comment:
If you have no warning at all, you can even specify ``-W`` that will faild the build on warning :)